anomalib/utils/callbacks/visualizer_callback.py

```diff
@@ -96,6 +95,60 @@ def _add_images(
         if "local" in module.hparams.project.log_images_to:
             visualizer.save(Path(module.hparams.project.path) / "images" / filename.parent.name / filename.name)
 
+    def generate_visualizer(self, outputs):
```

Comment:
This is a nice use of yield. I think we should also add a return type to the method. Something like `-> Iterator[Visualizer]`